mackerel-agent.rb

```diff
@@ -18,12 +18,9 @@ class MackerelAgent < Formula
 
   def install
     if build.head?
-      ENV['GOPATH'] = buildpath/'.go'
-      mkdir_p buildpath/'.go/src/github.com/mackerelio'
-      ln_s buildpath, buildpath/'.go/src/github.com/mackerelio/mackerel-agent'
       system 'make', 'build'
       bin.install 'build/mackerel-agent'
-      etc.install 'mackerel-agent.conf'
+      etc.install 'mackerel-agent.sample.conf' => 'mackerel-agent.conf'
```

Comment:
`mackerel-agent.conf` itself does not exist in mackerel-agent repository. It's copied from `mackerel-agent.sample.conf` in `make crossbuild` task:

https://github.com/mackerelio/mackerel-agent/blob/68dc1cb7bb57612875894687c6a320fcbd5ef687/Makefile#L56

Reply:
So this was broken all the time. (☍﹏⁰)

Reply:
Yes. maybe nobody tried `brew install --HEAD` for years……